### ISSUE TITLE
Improve GUI

### ISSUE THREAD

**Issue by ashwini0529** [GDGVIT/pip-gui#6]

**Comment by checor** [comment]
In what ways do you want to improve the GUI? I have a little of knowledge of Qt, I can help if you want.

Maybe putting the console output in a console-like widget would look nicer.

**Comment by ashwini0529** [comment]
The app looks completely functional but UI always helps. 
Adding enhancements and placing things where they look perfect could be a great start.
@checor why don't you try out something from your side and share the screenshot? 😄

**Comment by checor** [comment]
Sure! I will try in the evening.

Seeing your screenshots, I assume this project is for Linux, or does it works on Windows too?

**Comment by ashwini0529** [comment]
@checor Are you on it?

**Comment by checor** [comment]
Hello,

Apologies for the delay.

I will try something in the night if you are ok with that.

**Comment by checor** [comment]
Hello,

Did you made the GUI using .ui files? It would be nice to have them. Or, should I edit the generated files?

**Comment by checor** [comment]
In case you don't have them, or you don't want to add it to the source, I've done the changes directly to the generated source. Will look like this:

![image](https://user-images.githubusercontent.com/3594530/31866547-c7d893d0-b746-11e7-8cde-06bc513af581.png)

Should I add a font to the resources system, or try to use a font the system had already instaled? I want to use a monospaced font.

**Comment by SubhashiniTatipamula** [comment]
The starting page gave me the gist of the package. After looking at the screen I felt the install/update/uninstall screens looked consistent except search option. Why not search on update/uninstall screens? Is the screen not same for all the 3 options with the dynamically changing buttons/data?

**Comment by ayushpriya10** [comment]
@SubhashiniTatipamula  You're quite right about the consistency and yes the search bar can surely be added to update/uninstall screens. I didn't add them when I wrote the program because the number of packages in these categories would be significantly less and hence didn't feel much need of a search segment. But if you wish, you can surely add it.

**Comment by Oliver-Habersetzer** [comment]
Just chiming in to add this:
On Windows with scaling enabled, the UI is unusable. This is 225%.
![image](https://github.com/GDGVIT/pip-gui/assets/42864929/62632293-7398-4032-abb2-464509c2d4f3)

At 100% it looks fine again ;)
![image](https://github.com/GDGVIT/pip-gui/assets/42864929/35465b7f-c549-4691-99fd-245d5af12c47)
![image](https://github.com/GDGVIT/pip-gui/assets/42864929/4278fe49-560d-4e7e-9fb9-b48e5b512424)
